mlflow/projects/__init__.py

```diff
@@ -195,6 +200,7 @@ def run(
     run_id=None,
     run_name=None,
     env_manager=None,
+    tracking_uri=None,  # Inserted at the bottom for backwards compatibility
```

Comment:
Fantastic! Thank you! 

```suggestion
    tracking_uri=None,
```